src/common/components/signUpActivationModal/signUpActivationModal.tpl.html

```diff
@@ -22,12 +22,32 @@ <h3 translate>Register Your Account</h3>
         </loading>
 
         <div ng-if="$ctrl.unverifiedAccount.status !== 'Active'">
-          <div class="row text-center">
+          <div class="row">
             <div class="col-xs-12">
-              <h4 translate>Activate Account</h4>
-              <p translate style="margin-bottom: 0px">You will need to go to your <b>email</b> and open an email with the subject</p>
-              <strong translate style="display: block; margin-bottom: 8px">Activate Account for Giving to Cru.</strong>
-              <p translate>Verify your account by opening a link that we will send in that email.</p>
+              <h4 translate>Please verify your email address.</h4>
+              <ul>
+                <li translate>You will need to check your email inbox for a message from us.</li>
+                <li>Open the email and locate the verification link provided.</li>
+                <li>Click on the verification link. This will typically be a button or a hyperlink.</li>
+                <li>You will then be redirected to Okta to create your account password.</li>
```

Comment:
These should also get `translate` properties in case they get translated at some point.